### ISSUE TITLE
subsetting issue with Seurat V3

### ISSUE THREAD

**Issue by comefromwater** [satijalab/seurat#1405]
Hello, can anybody help me with the subsetting issue with Seurat V3. I applied a criteria to subset my data, I have 6 datasets, the code works in 4 of them, but the last two datasets, I received the error message 
"Error: Under current subsetting parameters, the default assay will be removed. Please adjust subsetting parameters or change default assay"

The code I input "P5_fil<-subset(x=P5_fil,subset=Lum>0, invert=TRUE)" It is just simply subset cells do not express Lum gene

Thanks

Hui

**Comment by andrewwbutler** [comment]
Hi Hui,

Because you specified `invert = TRUE` in the `subset` call, this will take all cells where `Lum <= 0`. Likely what is happening is that all cells in that object express the Lum gene at some non-zero level.

**Comment by comefromwater** [comment]
Thanks for your reply. 

It took me a long time to figure this out. There are only a few cells express Lum gene, not every cells.

The problem is with the Seurat object I use. I have no idea why, but it seems like even though I updated Seurat object to the new version, the object doesn't function properly, it doesn't allow me to even normalize it.  
I have 6 batches of data. 4 batches were done in 10x Genomics V2, and the other two batches were done in 10x Genomic V3, it seems like our sequencing core also updated their cellranger to a new version.   I analyzed all of the data using Seurat V2, when I updated to Seurat V3, the first 4 batches done with 10xGenomics V2 works perfectly, but there are problems with the other two batches of data.  So, I created Seurat object from the raw matrice, it seems work out.
